website/contributing/release-branch-cut-and-rc0.md

```diff
@@ -63,11 +64,13 @@ Before continuing further, follow the [testing guide](/contributing/release-test
 
 Once you're done with the testing, you can kick-off the bump and publishing of RC0:
 
-```
+```bash
 # This will walk you through what version you are releasing
-./scripts/bump-oss-version.js --to-version 0.69.0-rc.0 --token <YOUR_CIRCLE_CI_TOKEN>
+yarn trigger-react-native-release --to-version 0.69.0-rc.0 --token <YOUR_CIRCLE_CI_TOKEN>
```

Comment:
```suggestion
yarn trigger-react-native-release --to-version 0.y.0-rc.0 --token <YOUR_CIRCLE_CI_TOKEN>
```

We use this format elsewhere, so we might as well can align here.

Reply:
the entire page is using 0.69 as the example version, so I'd rather keep it consistent across the page. Someone as a follow up PR (maybe in 6 months or so, to connect to your other comment) could do a pass on all the release pages and update them to change both the example code and remove "old versions" references. In quite a few pages we have things like "before 0.71" or even "before 0.68".
So maybe by the time 0.74 comes around (this Q4 or early next year) that would be a good time to give an overall pass to the whole section to remove old references.